docs/community/dscan-icu/index.md

```diff
@@ -0,0 +1,49 @@
+---
+search:
+  exclude: true
+
+title: Dscan ICU
+type: service
+description: DSCAN ICU is a tool for analyzing and sharing D-Scan results in EVE Online, supporting multiple languages.
+maintainer:
+  name: zifox666 Garveen
+  github: zifox666/eve-dscan-tool
```

Comment:
This is supposed to be maintainer/contact person link, not a link to the project itself, so I think just `zifox666`, not `zifox666/eve-dscan-tool` would be more appropriate.